sphinx/writers/latex.py

```diff
@@ -200,15 +200,10 @@ def __init__(self, language_code, use_polyglossia=False):
 
     def get_shorthandoff(self):
         # type: () -> unicode
-        shortlang = self.language.split('_')[0]
-        if shortlang in ('de', 'ngerman', 'sl', 'slovene', 'pt', 'portuges',
-                         'es', 'spanish', 'nl', 'dutch', 'pl', 'polish', 'it',
-                         'italian', 'pt-BR', 'brazil'):
-            return '\\ifnum\\catcode`\\"=\\active\\shorthandoff{"}\\fi'
-        elif shortlang in ('tr', 'turkish'):
-            # memo: if ever Sphinx starts supporting 'Latin', do as for Turkish
-            return '\\ifnum\\catcode`\\=\\string=\\active\\shorthandoff{=}\\fi'
-        return ''
+        return ('\\ifdefined\\shorthandoff\n'
+                '  \\ifnum\\catcode`\\=\\string=\\active\\shorthandoff{=}\\fi\n'
+                '  \\ifnum\\catcode`\\"=\\active\\shorthandoff{"}\\fi\n'
+                '\\fi')
```

Comment:
Awesome! In master branch, we can embed this to template string (`BEGIN_DOC`) directly. `shorthandoff` variable is no longer needed.

Reply:
Thanks for quick review @tk0miya I will make PR on master branch afterwards. Now merging into 1.7 branch.